portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageServiceImpl.java

```diff
@@ -503,6 +503,12 @@ public String getGroupMessagesRSS(
 			name = message.getSubject();
```

Comment:
@austinskim123 @mbowerman Does this if block make sense to you guys? If I add multiple message board posts, then the title of the RSS feed is the oldest post's title. This logic makes sense when performing ```getThreadMessagesRSS```, but I feel like we should always use the Group's name and description in this instance. Let me know what you think.

Reply:
@jonathanmccann , I'm not able to test right now because I'm currently running 'ant all' but assuming that it is the oldest post's title, I think it'd be most appropriate to do either what you're suggesting and always use the Group's name and description, or change it so that it gets set to the most recent post's title. Perhaps after my bundle is up and running again, I can test it and see. Thanks for the feedback!

Reply:
I'll go ahead and utilize the group name and description for these methods and see what the component owner thinks. Thanks.

Reply:
Thanks.